packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -159,15 +183,13 @@ export async function enqueueLinksByClickingElements(options) {
  * Clicks all elements of given page matching given selector.
  * Catches and intercepts all initiated navigation requests and opened pages.
  * Returns a list of all target URLs.
- *
- * @param {object} options
- * @param {Page} options.page
- * @param {string} options.selector
- * @param {number} [options.waitForPageIdleMillis]
- * @param {number} [options.maxWaitForPageIdleMillis]
- * @ignore
```

Comment:
keep the `@ignore` comment, that is for typedoc so it is not part of the API documentation